From 8ee3dfa698181f737b18e7facb5e6c2834669c40 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Wed, 17 Oct 2012 12:56:43 -0400
Subject: Remove unnecessary overrides in gnukfreebsd and gnuknetbsd hints.

hints/gnukfreebsd.sh and hints/gnuknetbsd.sh unconditionally
override Configure's values for ccdlflags and lddlflags, even though
the default Configure guesses should be correct for those systems.
Configure was altered in commit fb2e1bc0638d5a5d7ac552a79a71a996a5d604cc
(Perforce change 23909) to get the correct values, but later commit
46c947e8b9def6de34ac831834a3c290ab266515 (Perforce change 24017) included
these now-outdated hints file changes as part of importing a larger set
of patches from Debian.

This patch removes the unnecessary overrides.  Thanks to Niko Tyni for
digging up the history and supplying the hints/gnukfreebsd.sh patch.

Origin: upstream, http://perl5.git.perl.org/perl.git/commit/7dc65651902f0390dfb92783b32c0b4976885475
Bug: http://rt.perl.org/rt3/Ticket/Display.html?id=115324
Bug-Debian: http://bugs.debian.org/689713
Patch-Name: fixes/kfreebsd-overrides.diff
---
 hints/gnukfreebsd.sh |    3 ---
 hints/gnuknetbsd.sh  |    3 ---
 2 files changed, 6 deletions(-)

diff --git a/hints/gnukfreebsd.sh b/hints/gnukfreebsd.sh
index 1225f69..435afe7 100644
--- a/hints/gnukfreebsd.sh
+++ b/hints/gnukfreebsd.sh
@@ -5,6 +5,3 @@
 
 . ./hints/linux.sh
 
-# Configure sets these where $osname = linux
-ccdlflags='-Wl,-E'
-lddlflags='-shared'
diff --git a/hints/gnuknetbsd.sh b/hints/gnuknetbsd.sh
index 6ee1433..008547f 100644
--- a/hints/gnuknetbsd.sh
+++ b/hints/gnuknetbsd.sh
@@ -5,6 +5,3 @@
 
 . ./hints/linux.sh
 
-# Configure sets these where $osname = linux
-ccdlflags='-Wl,-E'
-lddlflags='-shared'
